components/formats-api/src/loci/formats/FormatReader.java

```diff
@@ -884,9 +884,8 @@ public boolean isInterleaved(int subC) {
       throw new FormatException("Image plane too large. Only 2GB of data can " +
         "be extracted at one time. You can workaround the problem by opening " +
         "the plane in tiles; for further details, see: " +
-        "http://www.openmicroscopy.org/site/support/faq/bio-formats/" +
-        "i-see-an-outofmemory-or-negativearraysize-error-message-when-" +
-        "attempting-to-open-an-svs-or-jpeg-2000-file.-what-does-this-mean", e);
+        "http://www.openmicroscopy.org/site/support/bio-formats/about/" +
+        "bug-reporting.html", e);
```

Comment:
Ah, OK, that makes sense.  Adding the anchor might be nice to help reduce any confusion.